src/models/Task.ts

```diff
@@ -1 +1,11 @@
-export interface Task {}
+import { Timestamp } from 'firebase/firestore/lite';
```

Comment:
```suggestion
import { Timestamp } from 'firebase/firestore';
```
The `firebase/firestore/lite`  is a bit different library, more lightweight but with a limited functionality. We use the `firebase/firestore` one